pkg/synthetictests/event_junits.go

```diff
@@ -41,6 +42,7 @@ func SystemUpgradeEventInvariants(events monitorapi.Intervals, duration time.Dur
 	tests = SystemEventInvariants(events, duration, kubeClientConfig)
 	tests = append(tests, testContainerFailures(events)...)
 	tests = append(tests, testDeleteGracePeriodZero(events)...)
+	tests = append(tests, testKubeApiserverProcessOverlap(events)...)
```

Comment:
this looks like a systemeventinvariant to me.  no matter what, we shouldn't have two processes running.

Reply:
@deads2k it exists in both